core/src/main/java/org/jruby/RubyGC.java

```diff
@@ -67,6 +67,32 @@ public static RubyModule createGCModule(ThreadContext context) {
         return defineModule(context, "GC").defineMethods(context, RubyGC.class);
     }
 
+    @JRubyMethod(module = true, visibility = PRIVATE)
+    public static IRubyObject config(ThreadContext context, IRubyObject recv) {
+        return RubyHash.newHash(
+            context.runtime,
+            RubySymbol.newSymbol(context.runtime, "implementation"),
+            RubyString.newString(context.runtime, "java")
+        );
+    }
+
+    @JRubyMethod(module = true, visibility = PRIVATE)
```

Comment:
You are likely right about `meta = true` here. Probably cleaner to do it as a separate PR.